src/ts/typings/index.ts

```diff
@@ -0,0 +1,31 @@
+export interface IDate {
```

Comment:
`I` > `Interface`, `T` > `Type` etc., to keep namespace different than components (`Date`, `Location` etc.)